omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/MatchingPatientsFragmentController.java

```diff
@@ -122,7 +122,7 @@
             for (Field biometricField : biometricFields.keySet()) {
                 BiometricSubject subject = biometricFields.get(biometricField);
                 List<BiometricMatch> biometricMatches = biometricEngine.search(subject);
-                if (biometricMatches.size() > 0) {
+				if (!biometricMatches.isEmpty()) {

```

Comment:
```suggestion
                if (!biometricMatches.isEmpty()) {

```
It is common in java files to use tabs as line indentation characters, and you did that well. However, it is worth keeping a consistent style for the whole file, if the file uses spaces instead of tabs and there are tabs on one line, this causes inconsistency.

The author of this code snippet used spaces, so even if it is against the convention, spaces should also be included in this line.

Of course, you can also correct the whitespace throughout the file to make it follow convention, but I'm afraid that goes beyond the scope of the ticket.